src/helpers/validations.js

```diff
@@ -61,17 +53,7 @@ function isValidRequiredField(validator, field) {
  * Date Validations *
  */
 function isValidYear(value) {
-  return (
-    Number(value) >= 1900 && Number(value) <= moment().add(100, 'year').year()
-  );
-}
-
-function isValidYearOrBlank(value) {
```

Comment:
Were these originally removed because you removed the `MonthYear` component?

Reply:
Yup - that was the only place where they were used.

Reply:
There are a lot of removed functions in here. Should they all still be removed now that you've added `MonthYear` back in?

Reply:
Yeah. All of these functions were removed before I removed `MonthYear`:

![image](https://user-images.githubusercontent.com/2008881/109334417-8b33b480-7815-11eb-9311-44440231e8a3.png)

This isn't super pressing to merge, so we can talk about it next week when we have more time.